### DISCUSSION TITLE
Support for EC2 Image Builder (EC2IB)

### DISCUSSION THREAD

**Discussion by awillis** [awslabs/aws-sdk-rust#91] [General]
As I understand it, support for the EC2 Image Builder service should not require support for the EC2 service. This service is an incredibly useful way to migrate bespoke internal build pipelines (packer based?) into AWS itself, with the ability to distribute to multiple AWS accounts.

**Comment by rcoh** [comment]
Can you file a "new service" issue? The imagebuilder service uses restJson1 so there's nothing blocking adding support for it

**Reply by awillis** [reply]
Is this the same as the 'feature request' issue template?

**Reply by rcoh** [reply]
ah, yep!
